skribenten-backend/.nais/dev.yaml

```diff
@@ -39,7 +39,7 @@ env:
   - name: PEN_SCOPE
     value: "api://dev-fss.pensjon-q2.pensjon-pen-q2/.default"
   - name: BREVBAKER_URL
-    value: "https://pensjon-brevbaker.intern.dev.nav.no"
+    value: "http://pensjon-brevbaker"
```

Comment:
Var dette med vilje?

Reply:
Ja, url'en må være slik for å få kontakt med brevbaker i dev. Med service discovery url'en: https://docs.nais.io/clusters/service-discovery/#short-names

Reply:
Ah. Vi er jo i samme namespace ja.